app/src/main/AndroidManifest.xml

```diff
@@ -146,6 +146,7 @@
                 <data android:pathPrefix="/embed/" />
                 <data android:pathPrefix="/watch" />
                 <data android:pathPrefix="/attribution_link" />
+                <data android:pathPrefix="/shorts" />
```

Comment:
Some items here have a trailing slash and some don't. Is there a meaningful difference between the two?

Reply:
Okay so the links seem to usually look like this ``https://youtube.com/shorts/xxxxxxxxxxx``.

It think there should be a ``/`` at the end as the actual video is identified by an url parameter.
The ``/watch`` endpoint e.g. uses query parameters ``?v=xyz`` that's why there isn't a ``/`` at the end.

```suggestion
                <data android:pathPrefix="/shorts/" />
```


<i><sub>This comment cost me at least 3 brain-cells whilst watching some "shorts". Is there some kind of warranty so I can get them back?</sub></i>

Reply:
Aha, that makes sense. A comment explaining this would be helpful; I'll open a separate PR for this.

> This comment cost me at least 3 brain-cells whilst watching some "shorts". Is there some kind of warranty so I can get them back?

<sub>Be thankful this is not TikTok we're talking about :P</sub>